CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+## 0.14.0 (September 9, 2024)
+
+NOTES:
+* **This release requires juju controller version 2.9.49 or later juju.**
+* This release uses juju client api code from the juju 3.5.1 release.
```

Comment:
Is there a reason we are using 3.5.1 and not 3.5.3?

Reply:
There was no change to the api code required for the PRs landed for this release. If 3.6 is out before the end of cycle, we'll update to use that.